README.md

```diff
@@ -102,6 +102,77 @@ Where the `format`s available are:
 - `spdx-json`: A JSON report conforming to the [SPDX 2.2 JSON Schema](https://github.com/spdx/spdx-spec/blob/v2.2/schemas/spdx-schema.json).
 - `table`: A columnar summary (default).
 
+## Private Registry Authentication
+
+Syft uses the [authn](https://github.com/google/go-containerregistry/tree/main/pkg/authn) library to solve for private registry access. 
+
+The config file listed in the linked authn library is where your credentials are stored when authenticating with private registries via some command like `docker login`.
+
+An example plain text `config.json` looks something like this:
+```
+// config.json
+
+{
+	"auths": {
+		"registry.example.com": {
+			"username": "AzureDiamond",
+			"password": "hunter2"
+		}
+	}
+}
+```
+
+If you want to see the example detailed simply via podman you can use this command to verify a single run:
+
+`podman run -v ./config.json:/tmp/config.json -e "DOCKER_CONFIG=/tmp" anchore/syft:latest  <private_image>`
+
+The below section shows a simple workflow on how to mount this config as a secret into a container on kubernetes.
+1. Create a secret. The value of `config.json` is important and it refers to the specification outlined [here](https://github.com/google/go-containerregistry/tree/main/pkg/authn#the-config-file).
+
+Here is the `secret.yaml` file that the pod configuration will consume as a volume.
+```
+# secret.yaml
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: registry-config
+  namespace: syft
+data:
+  config.json: <base64 encoded config.json>
+```
+
+`kubectl apply -f secret.yaml`
+
+2. Create your pod running syft. The env `DOCKER_CONFIG` is important because it advertises to the `authn` library where to look for the credential file. In the below example, setting `DOCKER_CONFIG=/config` informs the library that credentials can be found at `/config/config.json`. This is why we used `config.json` as the key for our secret. When mounted into containers the secrets' key is used as the filename.
+
+The `voluemMounts` section mounts our secret to `/config`. The `volumes` section names our volume and sources the secret we created in step one as the file we will be mounting.
```

Comment:
spelling: `voluemMounts`

Reply:
Might be good to use a spellcheck overall (GoLand has one that I use)

Reply:
it advertises 68 errors currently with typos in word syft, sbom, grype, podman, etc. voluemMounts was not caught because of the backticks.
![Screen Shot 2021-11-02 at 10 48 40 AM](https://user-images.githubusercontent.com/32073428/139870672-c2952a5d-8f13-4997-a0c4-68d430441cfe.png)